nautobot_chatops/dispatchers/webex.py

```diff
@@ -13,26 +13,32 @@
 logger = logging.getLogger("rq.worker")
 
 # Create a metric to track time spent and requests made.
-BACKEND_ACTION_LOOKUP = backend_action_sum.labels("webex_teams", "platform_lookup")
-BACKEND_ACTION_MARKDOWN = backend_action_sum.labels("webex_teams", "send_markdown")
-BACKEND_ACTION_BLOCKS = backend_action_sum.labels("webex_teams", "send_blocks")
-BACKEND_ACTION_SNIPPET = backend_action_sum.labels("webex_teams", "send_snippet")
+BACKEND_ACTION_LOOKUP = backend_action_sum.labels("webex", "platform_lookup")
+BACKEND_ACTION_MARKDOWN = backend_action_sum.labels("webex", "send_markdown")
+BACKEND_ACTION_BLOCKS = backend_action_sum.labels("webex", "send_blocks")
+BACKEND_ACTION_SNIPPET = backend_action_sum.labels("webex", "send_snippet")
 
 # pylint: disable=abstract-method
 
 
-class WebExTeamsDispatcher(AdaptiveCardsDispatcher):
-    """Dispatch cards and messages to WebEx Teams."""
+class WebExDispatcher(AdaptiveCardsDispatcher):
+    """Dispatch cards and messages to WebEx."""
 
-    platform_name = "WebEx Teams"
-    platform_slug = "webex_teams"
+    platform_name = "WebEx"
+    platform_slug = "webex"
 
     platform_color = "6EBE4A"
 
     def __init__(self, *args, **kwargs):
-        """Init a WebExTeamsDispatcher."""
+        """Init a WebExDispatcher."""
         super().__init__(*args, **kwargs)
-        self.client = WebexTeamsAPI(access_token=settings.PLUGINS_CONFIG["nautobot_chatops"]["webex_teams_token"])
+        # v1.4.0 Deprecation warning
+        if "webex_token" in settings.PLUGINS_CONFIG["nautobot_chatops"]:
+            access_token = settings.PLUGINS_CONFIG["nautobot_chatops"]["webex_token"]
+        else:
+            access_token = settings.PLUGINS_CONFIG["nautobot_chatops"]["webex_teams_token"]
+            logger.warning("The 'webex_teams_token' setting is deprecated, please use 'webex_token' instead.")
```

Comment:
```suggestion
        if (
            "webex_teams_token" in settings.PLUGINS_CONFIG["nautobot_chatops"]
            and "webex_token" not in settings.PLUGINS_CONFIG["nautobot_chatops"]
        ):
            access_token = settings.PLUGINS_CONFIG["nautobot_chatops"]["webex_teams_token"]
            logger.warning("The 'webex_teams_token' setting is deprecated, please use 'webex_token' instead.")
        else:
            access_token = settings.PLUGINS_CONFIG["nautobot_chatops"]["webex_token"]
```

Reply:
Added this in with similar exception handling added like in the previous suggestions.